<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,7">
    <item>
     <widget class="QGroupBox" name="leftBar">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,1">
       <item>
        <widget class="QGroupBox" name="settingGrp">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>设定</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,3,6">
          <item>
           <widget class="QGroupBox" name="residentGrp">
            <property name="maximumSize">
             <size>
              <width>253</width>
              <height>247</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="residentLbl">
               <property name="text">
                <string>驻留集大小</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="buddy">
                <cstring>residentTxt</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="residentTxt">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>3</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="timeSet">
            <property name="title">
             <string>时间</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGroupBox" name="acMemTimeGrp">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="acMemTimeLbl">
                  <property name="text">
                   <string>访问内存时间</string>
                  </property>
                  <property name="buddy">
                   <cstring>acMemTimeTxt</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="acMemTimeTxt">
                  <property name="text">
                   <string>50</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="nsLbl_1">
                  <property name="text">
                   <string>ns</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="acTLBTimeGrp">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="acTLBTimeLbl">
                  <property name="text">
                   <string>访问快表时间</string>
                  </property>
                  <property name="buddy">
                   <cstring>acTLBTimeTxt</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="acTLBTimeTxt">
                  <property name="text">
                   <string>5</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="nsLbl_2">
                  <property name="text">
                   <string>ns</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="intrptTimeGrp">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="intrptTimeLbl">
                  <property name="text">
                   <string>中断处理时间</string>
                  </property>
                  <property name="buddy">
                   <cstring>intrptTimeTxt</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="intrptTimeTxt">
                  <property name="text">
                   <string>1e4</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="nsLbl_3">
                  <property name="text">
                   <string>ns</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="addrGrp">
            <property name="title">
             <string>地址</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
             <item>
              <widget class="QGroupBox" name="addrManuGrp">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,5">
                <item>
                 <widget class="QGroupBox" name="addrManuHeaderGrp">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,1">
                   <item>
                    <widget class="QLabel" name="addrManuLbl">
                     <property name="text">
                      <string>地址序列</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="loadFileBtn">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="res.qrc">
                       <normaloff>:/icon/res/icon/file.svg</normaloff>:/icon/res/icon/file.svg</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>30</width>
                       <height>30</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="addrManuTxt"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="addrAutoGrp">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,2">
                <item>
                 <widget class="QGroupBox" name="addrCntGrp">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="addrCntLbl">
                     <property name="text">
                      <string>生成数量</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="buddy">
                      <cstring>addrCntTxt</cstring>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="addrCntTxt">
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::UpDownArrows</enum>
                     </property>
                     <property name="maximum">
                      <number>255</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="addrLimGrp">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_3" columnminimumwidth="1,3,1,3">
                   <item row="0" column="0">
                    <widget class="QLabel" name="addrLimLbl">
                     <property name="text">
                      <string>范围</string>
                     </property>
                     <property name="buddy">
                      <cstring>addrLowLimTxt</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="addrLowLimTxt">
                     <property name="text">
                      <string>00000000H</string>
                     </property>
                     <property name="maxLength">
                      <number>9</number>
                     </property>
                     <property name="dragEnabled">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="connectSymbLbl">
                     <property name="text">
                      <string>~</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QLineEdit" name="addrGreatLimTxt">
                     <property name="text">
                      <string>0000f000H</string>
                     </property>
                     <property name="maxLength">
                      <number>9</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="addrGenerBtn">
                  <property name="text">
                   <string>生成</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restartBtn">
         <property name="text">
          <string>重启</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="centralSec">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>进程</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="actionGrp">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="startBtn">
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="addThreadGrp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="addThreadBtn_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/res/icon/addThrdBtnIcon.svg</normaloff>:/icon/res/icon/addThrdBtnIcon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>350</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addThreadBtn_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/res/icon/addThrdBtnIcon.svg</normaloff>:/icon/res/icon/addThrdBtnIcon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>350</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="addThreadBtn_4">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/res/icon/addThrdBtnIcon.svg</normaloff>:/icon/res/icon/addThrdBtnIcon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>350</width>
              <height>350</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="addThreadBtn_1">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/res/icon/addThrdBtnIcon.svg</normaloff>:/icon/res/icon/addThrdBtnIcon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>350</width>
              <height>350</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>addThreadGrp</zorder>
      <zorder>actionGrp</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
